__testfixtures__/ember-qunit-codemod/subject.input.js

```diff
@@ -65,3 +65,15 @@ moduleFor('service:foo', {
 test('can use custom subject', function(assert) {
   let subject = this.subject();
 });
+
+moduleFor('service:foo', 'Unit | Service | Foo', {
+  unit: true,
+
+  beforeEach() {
+    this.service = this.subject();
+  }
+});
+
+test('can use service', function (assert) {
+  this.servicer.something();
```

Comment:
Typo

Reply:
crappo, I missed this. Will fix asap

Reply:
Submitted https://github.com/rwjblue/ember-qunit-codemod/pull/24 to fix